src/offset/local/windows.rs

```diff
@@ -68,7 +68,12 @@ struct LocalSysTime {
     inner: SYSTEMTIME,
     /// The offset value from UTC
     offset: i32,
-    // Denotes the +/- 400 shifted a year value from invalid to valid
+    // Denotes the multiple of 400 year intervals shifted to create a valid year value. The
```

Comment:
Great comment.  While you're at it, can this be a docstring, e.g. leading `///` ?